packages/core/src/VRange.ts

```diff
@@ -116,6 +116,23 @@ export class VRange {
         const startIndex = this.start.parent.childVNodes.indexOf(this.start);
         return this.startContainer.childVNodes[startIndex + 1] === this.end;
     }
+    /**
+     * Return true if the range is contained within the given container.
```

Comment:
Technically, this sentence is wrong. It returns true if at least the start or the end of the rage is contained within the given container. The *entire* range being contained within a given container is something else.

Reply:
_You_'re the end of the rage.

Reply:
https://media.giphy.com/media/3orieZbA0wPRjR0b8Q/giphy.gif